extensions/ql-vscode/src/remote-queries/variant-analysis-monitor.ts

```diff
@@ -64,6 +64,7 @@ export class VariantAnalysisMonitor {
       if (variantAnalysisSummary.scanned_repositories) {
         variantAnalysisSummary.scanned_repositories.forEach(scannedRepo => {
           if (!scannedReposDownloaded.includes(scannedRepo.repository.id) && scannedRepo.analysis_status === 'succeeded') {
+            void commands.executeCommand('codeQL.autoDownloadVariantAnalysisResult', scannedRepo);
```

Comment:
This only seems to pass the `scannedRepo` to the command. Should this also be passing the `variantAnalysisSummary` (since that is one of the arguments of this command), or will that automatically be included?

Reply:
That is totally a mistake. Thanks for catching it! We unfortunately don't write tests for these commands at the moment and I admit I am totally reliant on some test screaming at me about the right amount of params. Thanks for catching it!